tests/test_models.py

```diff
@@ -130,6 +130,34 @@ def test_match_pyrtools(self, curie_img, n_scales):
             # after upsampling up to one row/column. This causes inconsistency on the right and
             # bottom edges, so they are exluded in the comparison.
 
+    @pytest.mark.skipif(DEVICE.type == 'cpu', reason="Can only test on cuda")
```

Comment:
I gave it a try, let me know what you think

Reply:
I actually meant one single test that takes in the name of the model as a function parameter, instead of writing on function per model. Anyway, not a big problem.

Reply:
ah yeah, that makes much more sense. pushing that change and also testing they all pass my validation function.